keys_test.go

```diff
@@ -73,7 +70,7 @@ func TestImportKeys(t *testing.T) {
 	config.GpgFlags = fmt.Sprintf("--homedir %s --keyserver 127.0.0.1", keyringDir)
 
 	server := startPgpKeyServer()
-	defer server.Shutdown(nil)
+	defer server.Shutdown(context.TODO())
```

Comment:
Seems like it should have been `server` passed here all along. @qrwteyrutiyoup any reason it was nil?

Reply:
Sorry, I am currently with (very) limited Internet access. No good reason it was nil; I must have missed it. Thanks for the fix!